experimental/generation/generator/packages/library/src/dialogGenerator.ts

```diff
@@ -533,7 +533,7 @@ async function allFiles(root: string): Promise<Map<string, string>> {
 // publishing.
 async function generateSingleton(schema: string, inDir: string, outDir: string) {
     let files = await allFiles(inDir)
-    let mainName = `${schema}.main.dialog`
+    let mainName = `${schema}.dialog`
```

Comment:
Could you put this into the generator.lg file so it could be controlled from there like the other file name?